lib/doorkeeper/openid_connect/helpers/controller.rb

```diff
@@ -75,8 +75,7 @@ def handle_oidc_prompt_param!(owner)
             when 'consent'
               render :new
             when 'select_account'
-              # TODO: let the user implement this
-              raise Errors::AccountSelectionRequired
```

Comment:
Can you please remove the now unused `Errors::AccountSelectionRequired` in `lib/doorkeeper/openid_connect/errors.rb`?

Reply:
https://github.com/doorkeeper-gem/doorkeeper-openid_connect/pull/111/commits/ddc0ede949f40e745ed51c49f96826da6adb165f#diff-b2596ff7dd36df28fc1ab37344ff56a9L29
Removed it 🙆‍♂️